examples/advanced/README.md

```diff
@@ -16,3 +16,7 @@ The following resources are provisioned by this example:
 - A basic VPC and subnet.
 - A Context-based restriction (CBR) network zone containing the VPC.
 - CBR rules that allow only the VPC to access the Object Storage instance and buckets over the private endpoint.
+
+## Note:
+
+To run this example, you have to set the values for the environment variables being used in variables.tf or can pass them at the time of running the example.
```

Comment:
@ocofaigh Do we need to say anything? We do assume a certain level of Terraform knowledge here. 

As for Projects/Schematics, I assume you're thinking about what we need to show from the readme in the catalog details page.

We could be generic:

> To run this example, set the values for the inputs in the  `variables.tf` file.

But that doesn't work for projects or schematics because of the reference to the file.

The current UI in the catalog has the `Work with code` tab with instructions about how to download and then run Terraform commands, but doesn't talk about inputs. I can't remember whether that remains in the planned UI changes.

So I don't have a good answer.

---

Also, in general, we don't use `Note` callouts. We do sometimes use `:information_source: **Tip:**` and `:exclamation: **Important:**` for information that should be called out. This information doesn't need to be called out but could be just a paragraph.

Reply:
I propose we remove it all together